### PR TITLE
test

### PR DESCRIPTION
**Pull request by OleksandrFEdev**
- [DEMO LINK](https://OleksandrFEdev.github.io/layout_calendar/)
- [TEST REPORT LINK](https://OleksandrFEdev.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's to fix this
Іt is necessary that all the last days also have a proper beginning
image
Let's center the calendar in the center of the page horizontally and vertically
image

File: src/styles/utils/variables.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+$gap: 1px;
+$size-day: 100px;
+$calendar-padding: 10px;
+$month-lengths: 28, 29, 30, 31;
+
+$dayOfWeek:
+  mon 0,
+  tue 1,
+  wed 2,
+  tue 3,
```
Comment (volodymyr-soltys97) [inline src/styles/utils/variables.scss]: There is a duplicate 'tue' entry in the $dayOfWeek list. It should be 'thu' instead.

File: src/index.html [hunk 2]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="container">
+      <div class="calendar calendar--start-day--sun calendar--month-length--29">
+        <div class="calendar__cells"></div>
```
Comment (volodymyr-soltys97) [inline src/index.html]: The class for the individual cells should be 'calendar__cell' instead of 'calendar__cells' to follow BEM naming conventions.

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
But let's make your code more perfect

File: src/styles/utils/reset.scss [hunk 3]
```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+}
+
+html,
+body {
```
Comment (anastasiiavorobiova) [inline src/styles/utils/reset.scss:6]: Add 'margin: 0;' and 'padding: 0;' to the 'html, body' section to reset browser default values.

**Review by anastasiiavorobiova — APPROVED** [review]
Great work

**Pull request closed without merging**